src/styles/blocks/card.scss

```
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  padding-top: 32px;
+  padding-left: 16px;
+  padding-right: 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $border-card-color;
+
+  background-color: $background-card-color;
+
+  &__image {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    width: 100%;
+    margin-bottom: 4px;
+
+    color: $text-accent-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__number {
+    align-self: flex-start;
+
+    margin-bottom: 16px;
+
+    color: $text-secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__marks {
+    display: flex;
+    justify-content: space-between;
+
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: $text-accent-color;
+    text-align: right;
+    font-size: 10px;
+    line-height: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    width: 100%;
+    margin-bottom: 16px;
+
+    &-label {
+      color: $text-secondary-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $text-accent-color;
+      text-align: right;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 100%;
+    height: 40px;
+    border: 1px solid $button-accent-color;
+    border-radius: 5px;
+    background-color: $button-accent-color;
+
+    color: $background-card-color;
+
+    &:hover {
+      transition-duration: 0.7s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```